spec/models/reception/resource_factory_spec.rb

```diff
@@ -476,13 +478,15 @@
              name: 'compound_sample_3',
              external_id: 'uuid-1',
              species: 'human',
-             supplier_name: 'supplier_name_2'
+             supplier_name: 'supplier_name_2',
+             donor_id: 'donor-1'
            },
            {
              name: 'compound_sample_4',
              external_id: 'uuid-2',
              species: 'human',
-             supplier_name: 'supplier_name_2'
+             supplier_name: 'supplier_name_2',
+             donor_id: 'donor-1'
```

Comment:
I couldn’t find any test cases covering the scenario with different donor IDs in the data above. Would it be good to add?